spec/features/proposals_spec.rb

```diff
@@ -1410,7 +1410,7 @@
       end
     end
 
-    scenario "Order by relevance by default", :js do
+    scenario "Order by relevance by default", :spanish_search, :js do
```

Comment:
It looks like this test was passing by coincidence (our fault, not yours :wink:); otherwise it would work in English :thinking:. It also assumes the factory will include the word "what" somewhere. And there's a somewhat-similar-but-different test in the debates spec.

You can leave it like this :+1:. I'll see if we can find a better way to test for relevance and will open another pull request.